src/main/java/cr/pernix/dashboard/services/ManagerService.java

```diff
@@ -38,7 +38,7 @@ public static synchronized ManagerService getInstance() {
         transaction.begin();
         if (transaction.getStatus().equals(TransactionStatus.NOT_ACTIVE))
             LOGGER.debug(" >>> Transaction close.");
-        Query query = session.createQuery("from Manager");
+        Query query = session.createQuery("from Manager m where m.active = true");
```

Comment:
You should add another method to get the inactive users, and activate one of thouse. Someone could deactived a user by error